dapr/proto/runtime/v1/appcallback.proto

```diff
@@ -287,6 +290,18 @@ message TopicRule {
   string path = 2;
 }
 
+// BulkSubscribeRequest is the message to pass settings for bulk subscribe
+message BulkSubscribeRequest {
+  // Required. Flag to enable/disable bulk subscribe
+  string enabled = 1;
+
+  // Optional. Max number of messages to be sent in a single bulk request
+  string maxBulkSubCount = 2;
```

Comment:
maxMessagesCount since this property is already in BulkSubscribeRequest, having "BulkSub" is redundant.

Reply:
Done.